homeassistant/components/discord/notify.py

```diff
@@ -39,30 +39,53 @@ def __init__(self, hass, token):
 
         discord.VoiceClient.warn_nacl = False
         discord_bot = discord.Client(loop=self.hass.loop)
+        images = None
 
         if ATTR_TARGET not in kwargs:
             _LOGGER.error("No target specified")
             return None
 
+        if ATTR_DATA in kwargs:
+            data = kwargs.get(ATTR_DATA)
+
+            if ATTR_IMAGES in data:
```

Comment:
This appears to have caused an unintended breaking change. If `data` is not in the service call it throws a `NoneType` error here. At minimum you must have `data:{}` now in order for the service call to be successful.  Reported in issue #23948

Reply:
Yes I saw that only after I updated do 0.93, and was waiting for the weekend to fix it